stories/core_components/GeoPushpinChart.tsx

```diff
@@ -414,3 +418,76 @@ storiesOf("Core components/GeoPushpinChart/Config/Legend", module)
             ),
         ),
     );
+storiesOf("Core components/GeoPushpinChart/Config/ColorMapping", module)
```

Comment:
I think `.../Color` is enough since we have 3 three types of color here

Reply:
fixed

Reply:
fixed